### PR TITLE
Yoshi Server: copy Dockerfile

### PR DESCRIPTION
**Pull request by yanivefraim**
On CI, we want to create the Dockerfile in case of a Yoshi Server.

### CODE REVIEW

File: packages/yoshi-common/src/copy-docker.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import path from 'path';
+import fs from 'fs-extra';
+import { Config } from 'yoshi-config/build/config';
+
+export = async function async(config: Config, cwd = process.cwd()) {
```
Comment (ronami) [inline packages/yoshi-common/src/copy-docker.ts:5]: why not use `export default`?

File: packages/yoshi-common/src/copy-docker.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  if (!config.yoshiServer) {
+    return;
+  }
+  const projectDockerExists = fs.existsSync(path.resolve(cwd, 'Dockerfile'));
```
Comment (ronami) [inline packages/yoshi-common/src/copy-docker.ts:9]: I think we can add `Dockerfile` to https://github.com/wix/yoshi/blob/master/packages/yoshi-config/src/paths.ts.

File: packages/yoshi-common/src/templates/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+FROM docker-repo.wixpress.com/wix-bootstrap-onbuild:stable
+MAINTAINER {%authorName%} <{%authorEmail%}>
```
Comment (ronami) [inline packages/yoshi-common/src/templates/Dockerfile:2]: I think we'll need to fill these as we copy it to a project, normally it happens when the developer generates a project with CYA.

File: packages/yoshi-flow-app/src/scripts/build.ts [hunk 4]
```diff
@@ -79,6 +80,7 @@ const build: cliCommand = async function(argv, config) {
         clientProjectName: config.clientProjectName,
         staticsDir: config.clientFilesPath,
       }),
+      copyDocker.default(config),
```
Comment (ronami) [inline packages/yoshi-flow-app/src/scripts/build.ts:83]: we can use `export default` and then we won't have to use `.default`.

File: packages/yoshi-common/src/copy-docker.ts (hunk 2 above)
Comment (yanivefraim) [reply packages/yoshi-common/src/copy-docker.ts:9]: fixed, thanks

File: packages/yoshi-common/src/templates/Dockerfile (hunk 3 above)
Comment (yanivefraim) [reply packages/yoshi-common/src/templates/Dockerfile:2]: nice catch, done. 10x!

**Pull request merged**